persistence/persistence-mybatis-test/src/test/java/org/cbioportal/persistence/mybatis/MutationMyBatisRepositoryTest.java

```diff
@@ -145,7 +145,6 @@ public void getMutationsWithAll() throws Exception {
 
 		List<Mutation> result = mutationMyBatisRepository.getMutations(sampleIds, entrezGeneIds, 6);
 		Assert.assertEquals(1, result.size());
-		Assert.assertNotNull(result.get(0).getMutationEvent().getGene());
```

Comment:
I'm guessing the rationale for dropping this is that a mutation must be associated with a mutationEvent and a mutationEvent must have a Gene (not null) ?  so this test cannot possibly fail?